### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Agetuni**
Create single page application 
- Add Navigation bar
- Add copyright footer 
- Notify the user when book library data is empty

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -50,3 +50,71 @@ form button {
...
+  font-size: 15px;
+}
+
+.nav {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  float: right;
+}
```
Comment (mukhammadrizooff) [inline style/style.css:65]: - [ ] [REQUIRED] Good job so far! Please add `hover` and `active` effect to the navigation items as shown in the wireframe.

image

- ### Your design:

image

File: style/style.css [hunk 2]
```diff
@@ -50,3 +50,71 @@ form button {
...
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  border: none;
+}
+
+nav {
+  display: inline;
+}
```
Comment (mukhammadrizooff) [inline style/style.css]: - [ ] [REQUIRED] You should place a `vertical line` between each navigation item.

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Agetuni, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
-  No Linter errors ✔️
- Navigation working well ✔️  

## Required Changes ♻️
- [ ] [REQUIRED] This milestone should be done with the group but I don't see any changes that **@MrOmachi** has made. Please make the changes I request below with @MrOmachi.

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Agetuni** [comment]
@mukhammadrizooff   Mr. Omachi has done the major  task of the day and the middle commit was his.
I just fixed linters issue finally  
image

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**